src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const fullCost = 40 * days;
+  let discount = 0;
+
+  if (days >= 3 && days < 7) {
+    discount = 20;
```

Comment:
checklist 3

Reply:
where i could find the checklist. the link doesn't work https://mate-academy.github.io/style-guides/javascript-standard-modified

Reply:
done. thanks